doc/api/modules.md

```diff
@@ -529,34 +529,43 @@ const x = require('./x');
 console.log(x.a);
 ```
 
-#### exports alias
+#### exports shortcut
 <!-- YAML
 added: v0.1.16
 -->
 
-The `exports` variable that is available within a module starts as a reference
-to `module.exports`. As with any variable, if you assign a new value to it, it
-is no longer bound to the previous value.
+The `exports` variable that is available within a module starts as if
+`module.exports` had been assigned to it before your module was evaluated.
```

Comment:
Please avoid the use of informal pronouns such as `you`, `your` and `us` in the docs :-)